pkgs/development/tools/misc/itstool/default.nix

```diff
@@ -2,11 +2,11 @@
 # We need the same Python as is used to build libxml2Python
 
 stdenv.mkDerivation rec {
-  name = "itstool-2.0.4";
+  name = "itstool-2.0.2";
```

Comment:
Please add a comment that 2.0.4 is broken linking to the upstream issue, so that we do not accidentally update again.